popup.js

```diff
@@ -0,0 +1,359 @@
+
+    <div class="card-flex proArt card-type1 card4">
+        <div class="text-container card4-hidde">
+            <h3 class="card-title crete font-white desktop-display-none">
+                Profesional Art Printing Data
+            </h3>
+            <h3 class="card-title crete font-white display-none show-on-desktop">Profesional Art Printing
+                Data
+                More</h3>
+            <p class="card-description inter400 font-white">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="tag">
+                <li class="tag-description-2">HTML</li>
+                <li class="tag-description-2">Bootstrap</li>
+                <li class="tag-description-2">Ruby</li>
+            </ul>
+        </div>
+        <button type="button" class="see-prj inter700 wide desktop-display-none">
+            See project
+        </button>
+    </div>
+
+    <div class="card-flex proArt card-type2 card5">
+        <div class="text-container card5-hidde">
+            <h3 class="card-title crete font-white desktop-display-none">
+                Profesional Art Printing Data
+            </h3>
+            <h3 class="card-title crete font-white display-none show-on-desktop">Data Dashboard Healthcare
+            </h3>
+            <p class="card-description inter400 font-white">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="tag">
+                <li class="tag-description-2">HTML</li>
+                <li class="tag-description-2">Bootstrap</li>
+                <li class="tag-description-2">Ruby</li>
+            </ul>
+        </div>
+        <button type="button" class="see-prj inter700 wide desktop-display-none">
+            See project
+        </button>
+    </div>
+
+
+    <div class="card-flex proArt card-type3 card6">
+        <div class="text-container card6-hidde">
+            <h3 class="card-title crete font-white desktop-display-none">
+                Profesional Art Printing Data
+            </h3>
+            <h3 class="card-title crete font-white display-none show-on-desktop">Website Portfolio</h3>
+            <p class="card-description inter400 font-white">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="tag">
+                <li class="tag-description-2">HTML</li>
+                <li class="tag-description-2">Bootstrap</li>
+                <li class="tag-description-2">Ruby</li>
+            </ul>
+        </div>
+        <button type="button" class="see-prj inter700 wide desktop-display-none">
+            See project
+        </button>
+    </div>
+
+</div>
+</section>`;
```

Comment:
- Great job so far @Team :muscle: However, as the previous reviewer mentioned, the projects sections shouldn't be hardcoded. Good job removing the content from the html file but I noticed that you only replaced them in the javascript file using template literals.
- But as mentioned in the project requirements, the projects section needs to be created dynamically. :+1: I have attached a screenshot below.
![image](https://user-images.githubusercontent.com/102699958/231513024-d2bd2106-4953-4ce7-9436-9ec29140495c.png)

- [ ] This means the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. :wink: 
- You can check an example of this student's project: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95
- So kindly use the information in the projects array and not hardcoded like you currently have.